waku/node.go

```diff
@@ -233,8 +233,11 @@ func Execute(options Options) {
 
 	if !options.Relay.Disable {
 		for _, nodeTopic := range options.Relay.Topics {
-			_, err := wakuNode.Relay().SubscribeToTopic(ctx, nodeTopic)
+			sub, err := wakuNode.Relay().SubscribeToTopic(ctx, nodeTopic)
```

Comment:
Why is this subscription need if it's not read?

Reply:
There is an alternative implementation where I unregister the broadcaster subscription, but keep the underlying pubsub subscription. That should also work

Reply:
Actually, I'm going to do that. Slightly lower overhead. Tested and it works.